src/service/build-content-items.ts

```diff
@@ -41,7 +44,63 @@ export const mapPodcastsToContentItems = (
     return items.filter(item => item.title !== '');
 };
 
-export const mapVideosToContentItems = (
+const descriptionWithLinks = (description: string) => {
+    let newDescription = description;
+    const matches = description.match(/(https?:\/\/[^\s]+)/g);
+    matches?.forEach(match => {
+        const replaceText =
+            '<a target="_blank" rel="noreferrer">' + match + '</a>';
+        newDescription = newDescription.replace(match, replaceText);
+    });
+    return newDescription;
+};
+
+export const slugify = (text: string) => {
+    const slug = text.replace(/[^a-zA-Z0-9]+/g, '-');
+    return slug.toLowerCase();
+};
+
+export const mapMongoDBTVShowsToContentItems = async (
+    allMongoDBTVShows: MongoDBTVShow[],
+    isRuntime: boolean
+) => {
+    let mappedMongodbTVShows: ContentItem[] = [];
+    // we can leverage preval if it's runtime.
```

Comment:
@mcolella14 : what is the meaning of runtime ?

Reply:
As opposed to build time. If we call this method at build time, we cannot use preval data since it will not have been built, but if we call it while the app is running (like if a user makes a request and we generate a page server-side), we can use that preval data.